src/main/java/com/google/api/codegen/util/java/JavaNameFormatter.java

```diff
@@ -72,4 +72,9 @@ public String packageFilePathPiece(Name name) {
   public String classFileNameBase(Name name) {
     return name.toOriginal();
   }
+
+  @Override
+  public String humanName(Name name) {
+    throw new UnsupportedOperationException();
```

Comment:
Go ahead and provide the same implementation for all of the languages - I'm not sure there's a reason for it to be different.